### ISSUE TITLE
Can not pick up EDIM 1.3.4

### ISSUE THREAD

**Issue by alex-konovalov** [frankluebeck/EDIM#11]
Due to the following line:

https://github.com/frankluebeck/EDIM/blob/df8a4ed2cb3fcdaf8522e2750c65cd34b2dd28db/PackageInfo.g#L14

the following error occurs:
```
* EDIM - new version 1.3.4 discovered!!!
  ============================================
  Getting new archives from 
  http://www.math.rwth-aachen.de/~Frank.Luebeck/EDIM/EDIM-1.3.3
[ "-win.zip", ".tar.bz2", ".tar.gz" ]
  unpacking EDIM-1.3.3.tar.gz
  ERROR (EDIM): PackageInfo.g on web refers to version 1.3.4, but 
PackageInfo.g in the archive has version 1.3.3! SKIPPING!!!
```
Since this can not be fixed by only updating the package archive (you need also to fix the `PackageInfo.g` file), could you please make a new release and increment the version number (1.3.5 would be OK) so that package update scripts will be able to pick it up?

You could also consider using https://github.com/gap-system/ReleaseTools and https://github.com/gap-system/GitHubPagesForGAP which allow to automate publishing a release on GitHub pages and have a number of benefits (releases recognised and properly treated by GitHub, https access instead of http etc.). If you don't consider that at the moment, perhaps you can find the snippet from `PackageInfo.g` at https://github.com/gap-system/ReleaseTools to be useful - it reuses components of `PackageInfo.g` to form a number of other components automatically, reducing the risk of errors when one should update the same text in several places:

```
SourceRepository := rec(
    Type := "git",
    URL := Concatenation( "https://github.com/gap-packages/", ~.PackageName ),
),
IssueTrackerURL := Concatenation( ~.SourceRepository.URL, "/issues" ),
PackageWWWHome  := Concatenation( "https://gap-packages.github.io/", ~.PackageName ),
README_URL      := Concatenation( ~.PackageWWWHome, "/README" ),
PackageInfoURL  := Concatenation( ~.PackageWWWHome, "/PackageInfo.g" ),
ArchiveURL      := Concatenation( ~.SourceRepository.URL,
                                 "/releases/download/v", ~.Version,
                                 "/", ~.PackageName, "-", ~.Version ),
```
and adapt it to your setup.

**Comment by fingolfin** [comment]
It would also be nice if the release commits were tagged -- there are tags v1.3.2 and v1.3.3, but not v1.3.4.

**Comment by alex-konovalov** [comment]
Thanks @fingolfin (Pinging @frankluebeck to help to discover this issue)

**Comment by frankluebeck** [comment]
Thanks for spotting this.

I have corrected a typo in the release script, released 1.3.5 and pushed my tags to the public repository.

**Comment by alex-konovalov** [comment]
Thank you @frankluebeck - it worked this time. 

Also note that you can update your release script to omit `.gitignore` file.

```
* EDIM - new version 1.3.5 discovered!!!
  ============================================
  Getting new archives from 
  http://www.math.rwth-aachen.de/~Frank.Luebeck/EDIM/EDIM-1.3.5
[ "-win.zip", ".tar.bz2", ".tar.gz" ]
  unpacking EDIM-1.3.5.tar.gz
  Validation of the info file successful!
  Package EDIM 1.3.5 from 13/08/2019 has dependencies:
  * NeededOtherPackages [ [ "GAPDoc", ">= 1.5" ] ]
  * GAP 4.5
  * ExternalConditions 
[ "UNIX for the kernel function 'ElementaryDivisorsPPartRkExpSmall'" ]
  finding text files  . . .

=====================text files==========================
EDIM-1.3.5/CHANGES
EDIM-1.3.5/read.g
EDIM-1.3.5/VERSION
EDIM-1.3.5/GPL
EDIM-1.3.5/PackageInfo.g
EDIM-1.3.5/TODO
EDIM-1.3.5/init.g
EDIM-1.3.5/makedocrel.g
EDIM-1.3.5/README
EDIM-1.3.5/lib/hmmlll.gd
EDIM-1.3.5/lib/edim.gi
EDIM-1.3.5/lib/hmmlll.gi
EDIM-1.3.5/lib/util.gi
EDIM-1.3.5/lib/util.gd
EDIM-1.3.5/lib/edim.gd
EDIM-1.3.5/lib/util2.gi
EDIM-1.3.5/tst/test.g
EDIM-1.3.5/tst/edim.tst
EDIM-1.3.5/doc/lefttoc.css
EDIM-1.3.5/doc/chooser.html
EDIM-1.3.5/doc/chapBib_mj.html
EDIM-1.3.5/doc/chapInd_mj.html
EDIM-1.3.5/doc/mathml.css
EDIM-1.3.5/doc/ragged.css
EDIM-1.3.5/doc/chap0_mj.html
EDIM-1.3.5/doc/chap1.txt
EDIM-1.3.5/doc/toggless.css
EDIM-1.3.5/doc/chap1_mj.html
EDIM-1.3.5/doc/testexamples.g
EDIM-1.3.5/doc/edim.bib
EDIM-1.3.5/doc/nocolorprompt.css
EDIM-1.3.5/doc/chapBib.html
EDIM-1.3.5/doc/chapBib.txt
EDIM-1.3.5/doc/chap0.txt
EDIM-1.3.5/doc/chap0.html
EDIM-1.3.5/doc/chapInd.html
EDIM-1.3.5/doc/times.css
EDIM-1.3.5/doc/main.xml
EDIM-1.3.5/doc/edim.xml
EDIM-1.3.5/doc/chapInd.txt
EDIM-1.3.5/doc/manual.css
EDIM-1.3.5/doc/chap1.html
EDIM-1.3.5/doc/edim.tex
EDIM-1.3.5/src/compstat_with_ediv.c
EDIM-1.3.5/src/ediv.c

=====================end of the list of text files=======
=====================binary files========================
EDIM-1.3.5/Makefile.in
EDIM-1.3.5/Makefile.in.old
EDIM-1.3.5/configure
EDIM-1.3.5/configure.old
EDIM-1.3.5/tst/mat
EDIM-1.3.5/tst/mat2
EDIM-1.3.5/doc/toggless.js
EDIM-1.3.5/doc/manual.js
EDIM-1.3.5/doc/rainbow.js
EDIM-1.3.5/doc/edim.toc
EDIM-1.3.5/doc/edim.ind
EDIM-1.3.5/doc/edim.pnr
EDIM-1.3.5/doc/clean
EDIM-1.3.5/doc/manual.pdf
EDIM-1.3.5/doc/manual.six
EDIM-1.3.5/doc/edim.idx
EDIM-1.3.5/doc/edim.bbl

=====================end of the list of binary files=====
=====================ignored files=======================
EDIM-1.3.5/.gitignore

=====================end of the list of ignored files====
```
